usmqe_tests/api/conftest.py

```diff
@@ -36,3 +36,32 @@ def invalid_session_credentials(request):
     invalid_token = request.param
     auth = TendrlAuth(token=invalid_token, username=username)
     return auth
+
+
+@pytest.fixture
+def cluster_reuse(valid_session_credentials):
```

Comment:
Ok, sounds good. I would later propose that there will be more tests for cluster import. Each with a special `gdeploy` configuration in fixture so we can call from CI during installation (it would be optional) something like:
`pytest import_test_with_configuration_123`